server/src/completion.ts

```diff
@@ -102,16 +104,16 @@ const completionList = (output: string): ICompletionItem[] =>
  * Build Clang shell command
  */
 const buildCommand =
-  (userFlags: string[], position: Position, languageId: string) =>
+  (userFlags: string[], position: Position, doc: TextDocument) =>
     [
       'clang',
-      '-cc1',
-      ...userFlags,
       '-fsyntax-only',
-      languageId === 'c' ? '-xc' : '-xc++',
-      '-code-completion-macros',
-      '-code-completion-at',
-      `-:${position.line + 1}:${position.character + 1}`
+      ...userFlags,
+      `-I${path.dirname(Uri.parse(doc.uri).fsPath)}`,
```

Comment:
Why do you add to include folders the folder of the current file ?
I'm not sure of the interest here.